projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -957,38 +956,33 @@ function ( $site_purchase ) {
 	return empty( $bundle_purchases );
 }
 
-/**
- * Determines whether or not the Install the mobile app task should be visible.
- *
- * @return bool True if the Install the mobile app task should be visible.
- */
-function wpcom_launchpad_is_mobile_app_installed_visible() {
-	$is_atomic_site = ( new Automattic\Jetpack\Status\Host() )->is_woa_site();
-	// If the site is an Atomic site, we should not show the task.
-	if ( $is_atomic_site ) {
-		return false;
-	}
-
-	return true;
-}
-
 /**
  * Verifies if the Mobile App is installed for the current user.
  *
  * @return bool True if the Mobile App is installed for the current user.
  */
 function wpcom_launchpad_is_mobile_app_installed() {
```

Comment:
Quick note: it looks like the code to determine whether a task is complete will pass the completed task status to this function, which would allow us to skip remote calls or unnecessary checks altogether. That also suggests that we should mark the task as complete when we work out that the user has logged into the app before. That should prevent us from continually needing to make these API calls once the user has completed the task.
```suggestion
function wpcom_launchpad_is_mobile_app_installed( $is_complete ) {
	if ( $is_complete ) {
		return true;
	}

```

See below for the calling code:
https://github.com/Automattic/jetpack/blob/2f48ff68f3ade46e250800a313d40ef507ffd70e/projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php#L617

https://github.com/Automattic/jetpack/blob/2f48ff68f3ade46e250800a313d40ef507ffd70e/projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php#L464-L469

Reply:
Side note: if we do the above, we'll need to update the testing instructions to also reset the task completion status.